BasicSample/BasicSample/ChatViewControllers/BoldFormViewController.swift

```diff
@@ -0,0 +1,57 @@
+// ===================================================================================================
+// Copyright © 2020 bold360ai(LogMeIn).
+// Bold360AI SDK.
+// All rights reserved.
+// ===================================================================================================
+
+
+import UIKit
+import Bold360AI
+
+class BoldFormViewController: UIViewController {
+    var chatController: ChatController!
+    var formInfo: BrandedForm!
+    var formDelegate: BoldFormDelegate!
+
+    @IBOutlet weak var formTitle: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let formField = self.formInfo.form.formFields[0] as? BCFormField{
+            self.formTitle.text = formField.label
+        }
+    }
+    
+    @IBAction func submitTapped(_ sender: Any) {
+        (self.formInfo.form.formFields[0] as? BCFormField)?.label = self.formTitle.text
+        self.delegate.submitForm(self.form)
+    }
+    
+    override func viewDidDisappear(_ animated: Bool) {
+        self.delegate.submitForm(nil)
+        super.viewDidDisappear(animated);
+    }
+}
+
+    
+    
+extension BoldFormViewController: BoldForm {
```

Comment:
I'm not sure if it will work.
This solution is taken from the documentation we currently have. (https://github.com/Logmein-Bold-Mobile/bold360-mobile-docs-ios/blob/master/PresentFormsIOS.md)
Don't you think it's there on purpose?